airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -46,6 +46,8 @@ class GoogleAds:
     DEFAULT_PAGE_SIZE = 1000
 
     def __init__(self, credentials: Mapping[str, Any], customer_id: str):
+        if "login_customer_id" in credentials and not credentials["login_customer_id"].strip():
```

Comment:
if login_customer_id filed is empty then remove it?

Reply:
yep